### ISSUE TITLE
Open vs Closed Issues

### ISSUE THREAD

**Issue by RichardLitt** [ipfs/faq#99]
Currently, we have some closed issues. I don't think this is the right thing to do. All issues should be open, permanently. 

I propose we unclose all issues, so it is easier to search. Issues that are 100% answered can be locked.

**Comment by RichardLitt** [comment]
Also, added a meta label.

**Comment by ghost** [comment]
Generally agree, but I think we should still close duplicates, just to avoid clutter

**Comment by RichardLitt** [comment]
CC @diasdavid @whyrusleeping @noffle @haadcode @jbenet

**Comment by daviddias** [comment]
We should close duplicates and curate the content of the ones we keep.

**Comment by hackergrrl** [comment]
I agree with @diasdavid -- remove dupes but maintain a high quality control on the ones that we do leave open. I think promoting organic questions to _/proper FAQ questions and answers/_ as we go is really important.

**Comment by Kubuxu** [comment]
What do you think about deleting, moderating comments that are unclear out of topic for given issue (FAQ question)?

It might be controversial but if we want to keep high standard of the FAQ.

**Comment by hackergrrl** [comment]
@Kubuxu: if the question is poorly worded but not deduplicated, we could edit it to be more concise, and still provide a solid answer.

**Comment by RichardLitt** [comment]
I added an 'answered' label. #112 

Now I've gone through and put 'answered' on all of the issues I thought were pretty much done. Curating is _really hard_; but it is something we'll need to do. Closing duplicates, badly worded questions, and pointing to well-done answers is something I think we should do organically as we go along. At some point, I should write a doc that goes through and makes good answers for questions, too; that will probably be what happens to the FAQ section on the website (still forthcoming). 

Anyway, step in the right direction.
